src/App.tsx

```diff
@@ -83,11 +64,7 @@ export const App: React.FC = () => {
                 )}
 
                 {author && loaded && !hasError && posts.length > 0 && (
```

Comment:
consider moving long ternaries to a separate const with consistent name and use the const here